tests/unit_tests/blas/include/test_common.hpp

```diff
@@ -29,8 +29,8 @@
 #include <CL/sycl.hpp>
 
 namespace std {
-static cl::sycl::half abs(cl::sycl::half v) {
-    if (v < cl::sycl::half(0))
+static half abs(half v) {
+    if (v < half(0))
```

Comment:
Shouldn't this change be part of https://github.com/oneapi-src/oneMKL/pull/102?

Reply:
Sorry, I think this shouldn't be a part of any PR. I accidentally removed these with all the other `cl::sycl::` qualifiers from before the half type as asked for by @mmeterel

Reply:
@sbalint98 Can you remind why you needed to qualify half with cl::sycl:: namespaces to start with?

Reply:
In hipSYCL we define half inside that namespace, and if I am correct it is also required by the spack that half is defined in the sycl namespace. 

In table 138. in section 4.14.1 `half` is listed as 

> Additional scalar data types which are supported by SYCL within the sycl namespace